docs/usage.rst

```diff
@@ -374,6 +374,48 @@ Examples
     ...
 
 
+
+.. include:: usage/diff.rst.inc
+
+Examples
+~~~~~~~~
+::
+
+    $ mkdir testdir
+    $ echo asdf > testdir/file1
```

Comment:
if you cd into the testdir, you could avoid repeating it all the time (and give "." to borg create).